lib/botmetrics.rb

```diff
@@ -1,42 +1,54 @@
 require "json"
 require "excon"
 
-class BotMetrics
-  VERSION = "0.0.3"
+module BotMetrics
+  class Client
+    DEFAULT_API_HOST = "https://www.getbotmetrics.com".freeze
+
+    def initialize(api_key:, bot_id:, api_host: nil)
+      if api_key.nil? || api_key == ""
```

Comment:
Both `api_key` and `bot_id` should check for existence of environment variables and if only they are also blank should we raise an error